docs/USAGE.md

```diff
@@ -0,0 +1,276 @@
+
+## Creating a token
+
+Tokens are active by default and can easily be disabled from the tokens list.
+
+:::info
+Access to the "My API tokens" page is restricted to users with permission "Personal api tokens" as a "Server role". Adding "Personal api tokens" permission to "Live role" or "Edit role" is not sufficient for a user to create and manage tokens.
+:::
+
+To create a token:
+
+1. In Jahia, navigate to **Dashboard>My API tokens**.
+2. Click **Create Token**.
+3. In the Create a personal API token dialog, provide the following information:
+      - **Name**  
+        The name of the token is unique for each user. It is used only for token management purposes to identify the token’s purpose by the user. 
+      - **Expiration date**  
+        The expiry date is the date after which any calls to the API are deemed invalid. Once the expiry date is reached, tokens are not deleted and are not usable unless you update the expiration date to a later date.  
+     
+     ![Create Personal Api Token 1](./img/create-personal-api-token-01.webp)
+4. Click **Create**. The token displays on the page. This is **the only time** the token is visible to you. This token string is a sensitive piece of information. Make sure to keep it safe.
+
+   ![Create Personal Api Token 2](./img/create-personal-api-token-02.webp)
+5. To copy the token, click **Copy**. Then copy the token value to a safe location.
+6. Click **Close**.
+
+The token displays in the list of tokens on the **My API tokens** page.
+
+## Using tokens
+
+You can simply pass a token in the `authorization` header in any API call by prefixing it with `APIToken`.
+
+For example, the Apollo client would be instantiated like this in Typescript:
+
+```typescript
+export const apolloClient = (token: string): ApolloClient => {
+    return new ApolloClient({
+        link: new HttpLink({
+            uri: `JAHIA_URL/modules/graphql`,
+            headers: {
+              authorization: `APIToken ${token}`
+            }
+        }),
+        cache: new InMemoryCache(),
+        defaultOptions: {
+            query: {
+                fetchPolicy: 'no-cache',
+            },
+        },
+    })
+}
+```
+
+Or, to retrieve the result of the healthcheck, it would be like that:
+
+```bash
+curl -H "Authorization: APIToken XXXXXXXXXX" JAHIA_URL/modules/healthcheck
+```
+
+## Managing tokens
+
+:::info
+Access to the "My API tokens" page is restricted to users with permission "Personal api tokens"
+:::
+
+Tokens that you create display on the **My API tokens** page. The list shows information such as the name of the token, its access key, and expiration date. You can delete, activate and disable tokens from this page.
+
+![Managing Tokens](./img/managing-tokens-01.webp)
+
+:::infos
+**Note:** Tokens are only visible to users once when they are created. None of the information provided here is sufficient to perform an API call.
```

Comment:
What is meant by `None of the information provided here is sufficient to perform an API call.`

Does it related to the previous comment (if not, I'd just remove that part).